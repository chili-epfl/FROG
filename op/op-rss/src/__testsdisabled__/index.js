// NOTE: This is not working after the upgrade, should be fixed (problem is that Jest doesn't
// process with Babel properly

// import { operator } from '../index';

// jest.mock('frog-utils', () => {
//   const createUUID = () => {
//     let cntr = -1;
//     return () => {
//       cntr += 1;
//       return cntr;
//     };
//   };
//   const uuid = createUUID();
//   const frogutils = require.requireActual('frog-utils');
//   return { ...frogutils, uuid };
// });

// test('Correctly parses API request results with contents', () =>
//   expect(operator({ tag: 'reason' })).resolves.toMatchSnapshot());

// test('Correctly parses API request results returning empty', () =>
//   expect(operator({ tag: 'empty' })).resolves.toMatchSnapshot());
