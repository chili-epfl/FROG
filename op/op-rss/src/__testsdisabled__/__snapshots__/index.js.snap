// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly parses API request results returning empty 1`] = `
Object {
  "payload": Object {
    "all": Object {
      "config": Object {},
      "data": Array [],
    },
  },
  "structure": "all",
}
`;

exports[`Correctly parses API request results with contents 1`] = `
Object {
  "payload": Object {
    "all": Object {
      "config": Object {},
      "data": Array [
        Object {
          "content": "This is a great example of Kohonen's Self-organizing Maps  and the use of the U-Matrix. The authors were very thorough in explaining how it can be used.",
          "id": 0,
          "title": "Appendix. Technical demonstration of the SOMprocedure",
        },
        Object {
          "content": "Data Structure",
          "id": 1,
          "title": "The SOM consisted of 360 neurons on a 24by  15  map  grid,  with  hexagonal  lattice  andGaussian neighborhood  function.",
        },
        Object {
          "content": "This is the novel innovation approach to network analysis that I will be discussing in my article review presentation
",
          "id": 2,
          "title": "(Kohonen’s Self-Organizing Maps",
        },
        Object {
          "content": "I really like this concept and application of SNA with integration of critical discourse analysis (CDA), which is a course I am taking in the fall!! In another class, we read about critical quantitative approach, and I now think what makes it truly critical might be the integration of a mixed method approach, where something like ethnographic case studies or in this case discourse analysis is used in combination with SNA/statistical analyses to give it more robust findings.",
          "id": 3,
          "title": "Ryu, S., & Lombardi, D. (2015). Coding Classroom Interactions for Collective and Individual Engagement. Educational Psychologist, 50(1), 70–83. http://doi.org/10.1080/00461520.2014.1001891 (Note: An attempt to combine SNA with critical discourse analysis.)",
        },
        Object {
          "content": "These are important questions that need answered to truly understand complex social interactions.",
          "id": 5,
          "title": " address the questions in the 					previous paragraph",
        },
        Object {
          "content": "I am familiar with SPSS, but I had not considered using it for my final project until now. I am a little confused about how I would have to code the data (adjacency matrix) so that it works well in SPSS, but I think I'm going to try it just to see if I can get the same results that I obtain from using R.",
          "id": 15,
          "title": "analyses using 							these ego-level measures can be done using statistical packages like 								SPSS",
        },
        Object {
          "content": "I am interested in learning more about the NetDraw software in UCINET. While I like statistics, nothing really says SNA like having clear visuals that display network ties",
          "id": 16,
          "title": "UCINET is NetDraw, a visualization tool that has advanced 							graphic properties.",
        },
        Object {
          "content": "I need to more practice with creating cleaner graphs that show my network. So far, I've been sticking with the orange and blue colors but I plan to get a little more creative for the final project",
          "id": 17,
          "title": "visualizing your 						network in the form of a graph provides some hints as to whether your 							motivating research questions or hypotheses are worth pursuing or in 							need of revision.Figure",
        },
        Object {
          "content": "My project deviated from the author's process because I actually obtained the dataset before identifying research questions. I am using a public dataset so I had to develop me project around what I had available",
          "id": 18,
          "title": "Now that your network's boundary has been specified, you are ready to 						collect your data.",
        },
        Object {
          "content": "Throughout the semester, I have struggled with turning my research topic into a researchable question. I think I've come up with some decent questions, but I'm constantly rewording them.",
          "id": 19,
          "title": " First, start with an initial topic of interest and 						then turn this topic into a question. ",
        },
      ],
    },
  },
  "structure": "all",
}
`;
